examples/00-basic/01-basic_operators.py

```diff
@@ -1,3 +1,4 @@
+# noqa: D400
```

Comment:
@GuillemBarroso What is that skipping for?

Reply:
D400 indicates that the first line should end with a period, see [doc](http://www.pydocstyle.org/en/stable/error_codes.html).

@PProfizi, as we discussed, I will add the rule of ignoring D400 for the examples in the next PR, when I update the layout of the repo and I add the pre-commit.